internal_ws/yktrace/src/tir.rs

```diff
@@ -37,6 +37,7 @@ impl<'a, 'm> TirTrace<'a, 'm> {
         let mut rnm = VarRenamer::new();
         // Symbol name of the function currently being ignored during tracing.
         let mut ignore: Option<String> = None;
+        let mut ignore_count: usize = 0;
```

Comment:
Are there any `ignore` things that aren't `do_not_trace`?

Reply:
At the moment, no.

Reply:
Any function directly or indirectly called from a `#[do_not_trace]` function must also not be traced.

Reply:
Then I'd suggest this is the `do_not_trace_count` or similar (personally, I'd probably call it `dnt_trace` and add `# do_not_trace count` or similar).

Reply:
Try dfc8326.